apps/server/src/services/mcp-service/mcp.ts

```
@@ -0,0 +1,433 @@
+      refreshToken: connection.refreshToken,
+      email: connection.email,
+    },
+  });
+
+  return {
+    driver,
+    connectionId: connection.id,
+  };
+};
+
+export class ZeroMCP extends McpAgent<typeof env, {}, { phoneNumber: string }> {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)